fortinet_fortigate/plugin.spec.yaml

```
@@ -689,12 +689,24 @@ actions:
         description: Optional name to filter on
         type: string
         required: false
-        example: 198.51.100.100
+        example: MaliciousHost
+      subnet_filter:
+        title: Subnet Filter
+        description: Optional subnet to filter on
+        type: string
+        required: false
+        example: 198.51.100.100/32
+      fqdn_filter:
+        title: FQDN Filter
+        description: Optional FQDN to filter on
+        type: string
+        required: false
+        example: example.com
     output:
       address_objects:
         title: Address Objects
         description: A list of address objects
-        type: '[]object'
+        type: '[]address_objects'
```

```suggestion
        type: '[]address_object'
```
The type should be singular (update the type definition)